File: AngularESA/angularesa.client/src/app/Models/FlightsItinerary.ts
import { Carrier } from "./Carrier";
import { City } from "./City";

export interface FlightsItinerary {
  flightIATA: string;
  flightICAO: string;
  flightStatus: string;
  departureLocation: City;
  departureSchedule: Date;
  arrivalLocation: City;
  arrivalSchedule: Date;
  airline: Carrier;
}

File: PROJETOESA/Models/FlightsItinerary.cs
namespace PROJETOESA.Models
{
    public class FlightsItinerary
    {
        public string FlightIATA { get; set; }
        public string FlightICAO { get; set; }
        public string FlightStatus { get; set; }
        public City DepartureLocation { get; set; }
        public DateTime DepartureSchedule {  get; set; }
        public City ArrivalLocation { get; set; }
        public DateTime ArrivalSchedule { get; set; }
        public Carrier? Airline { get; set; }
    }
}
